.github/workflows/wheels.yml

```diff
@@ -214,6 +199,50 @@ jobs:
           CIBW="build; args: --no-isolation"
           echo "CIBW_BUILD_FRONTEND=$CIBW" >> "$GITHUB_ENV"
 
+      - name: Windows - set environment variables
+        if: matrix.buildplat[1] == 'win'
+        shell: pwsh
+        run: |
+          # Common Windows settings
+          $workspacePath = "${{ github.workspace }}".Replace("\","/")
+          echo "CIBW_ENVIRONMENT=PKG_CONFIG_PATH=$workspacePath" >> $env:GITHUB_ENV
+
+          # ARM64 specific settings
+          if ('${{ matrix.buildplat[1] }}' -eq 'win' -and '${{ matrix.buildplat[2] }}' -eq 'ARM64') {
+            echo "CC=clang-cl" >> $env:GITHUB_ENV
+            echo "CXX=clang-cl" >> $env:GITHUB_ENV
+            echo "FC=flang" >> $env:GITHUB_ENV
+            echo "TARGET_ARCH=${{ matrix.buildplat[2] }}" >> $env:GITHUB_ENV
+            echo "PATH=C:\vcpkg\installed\arm64-windows\tools\pkgconf;$($env:PATH)" >> $env:GITHUB_ENV
+          }
+
+      - name: Install rtools for win_amd64
+        if: ${{ matrix.buildplat[1] == 'win' && matrix.buildplat[2] == 'AMD64' }}
+        run: |
+          # mingw-w64
+          choco install rtools -y --no-progress --force --version=4.0.0.20220206
+          echo "c:\rtools40\ucrt64\bin;" >> $env:GITHUB_PATH
+
+      # can be remove after https://github.com/actions/partner-runner-images/issues/90
+      - name: Install LLVM for win_arm64
```

Comment:
Installing LLVM and `pkg-config` can now be done as a one-liner, invoking https://github.com/scipy/scipy/blob/main/.github/windows_arm64_steps/action.yml. As done here:

https://github.com/scipy/scipy/blob/fbb6ff839cfdf02304a3bd21d46a1189c01832e9/.github/workflows/windows.yml#L193-L194